### PR TITLE
[CNXC-292] Remove Privacy Level field Drop-down

### PR DESCRIPTION
**Pull request by colinywu**
### Problem Statement
The Privacy Level select field used for anonymization of data needs to be removed from the dashboard page.
Originally, the purpose of that setting was to anonymize personal data during public demos. 

### Implementation
Removed the Privacy Level drop-down from the PatientLookup component, (along with any of its dependent code), then adjusted styling of the patient look-up bar

### CODE REVIEW

**Review by boris-nguyen — APPROVED** [review]
Good work @colinywu

File: src/assets/scss/components/PatientLookup.scss [hunk 1]
```diff
@@ -3,22 +3,19 @@
...
     }
-    #toggle-Privacy-Level {
-      width: 1000px;
-    }
   }
 }
 
 .PatientLookupWrapper {
-  padding-bottom: 0px;
+  padding-bottom: 0em;
```
Comment (janakitti) [inline src/assets/scss/components/PatientLookup.scss:18]:
```suggestion
  padding-bottom: 0;
```
Units not needed when value is 0

**Review by janakitti — COMMENTED** [review]
Great work @colinywu! I would maybe suggest setting `width: 100%` for `form-display` to make sure the input field and button scale down when the screen width shrinks

**Review by janakitti — APPROVED** [review]
Looks great! :+1:

**Pull request merged**